java.yaml

```diff
@@ -4,9 +4,23 @@ steps:
 
 # build the user's source and generate artifacts for docker
 - name: 'gcr.io/gcp-runtimes/java/runtime-builder:latest'
-  args: ['--server-runtime', 'gcr.io/google_appengine/jetty:latest',
-  '--jar-runtime', 'gcr.io/google_appengine/openjdk:latest',
-  '--tomcat-runtime', 'gcr.io/google_appengine/tomcat:latest']
+  args:
+  # Mapping of server and JDK config names to runtime images. Underscores represent wildcards.
+  - '--server-runtimes-map'
+  - '{
+      "_#_"               : "gcr.io/google_appengine/jetty:9",
```

Comment:
This is a map with a compound key, and `#` is the separator between the jdk name and server name components. Maybe a pipe character would be clearer? I'm not sure that proper regex support is really necessary, but perhaps it would also be clearer to use `*` instead of `_` to represent wildcards. What do you think?